LinearRegression/Blog/main.tex

```diff
@@ -0,0 +1,235 @@
+\section*{Getting Started}
+\subsection*{Import Libraries}
+First, let's important all necessary libraries in the beginning to keep our code organized:
+\begin{figure}[H]\includegraphics[width=\linewidth]{1}\end{figure}
+\restoregeometry
+\subsection*{Read Dataset}
+For this analysis, I will use a real estate dataset that can be downloaded from \href{https://www.kaggle.com/quantbruce/real-estate-price-prediction}{here}. I choose to upload it to Google Colab from local drive. Of course, you can also replace the first argument in read\_csv() with the filepath or URL to the dataset that you'd like to use instead. The delimiter parameter is set to comma by default, but you can change it depending on the delimiter of your csv file.
+\begin{figure}[H]\includegraphics[width=\linewidth]{2}\end{figure}
+We can take a look at what the data looks like:
+\begin{figure}[H]\includegraphics[width=\linewidth]{3}\end{figure}
+\begin{figure}[H]\includegraphics[width=\linewidth]{4}\end{figure}
+As you can see, the first column in the real estate data set is redundant, so we should probably remove it. Otherwise, this data set seems directly usable. However, it's likely that your own data set will require more preprocessing--dealing with missing/invalid values, restructuring the data, and encoding dummy variables, etc.
+\begin{figure}[H]\includegraphics[width=\linewidth]{5}\end{figure}
+After data preprocessing, we can use the describe() function to obtain some summary statistics of our target variable, and the displot() function from the seaborn library to plot its distribution.
+\begin{figure}[H]\includegraphics[width=1.1\linewidth]{6}\end{figure}
+\begin{figure}[H]\centering\includegraphics[width=0.5\linewidth]{7}\end{figure}
+We see that the values of house price of unit area are normally distributed with perhaps a few outliers on the right.
+\section*{Checking Multicollinearity}
+Before the analysis, it is important to verify several regression conditions so as to make sure that our analysis is valid. Most of the conditions in linear regression can be checked easily with the residuals--the only snag is that we get the residuals only \textit{after} we perform the regression. Still, "no multicollinearity" is the condition that we are able to check beforehand, without using the residuals.
+
+Since a multiple regression involves more than one predictor variables, multicollinearity occurs when some of them are highly correlated with each other, which means each of these predictors will account for similar variance in the target variable. Therefore, though the presence of multicollinearity will not affect the predictive power of our model, it will make it more difficult for us to assess the individual influence of a predictor on our target variable. We can detect multicollinearity using either a correlation matrix or VIF factors.
```

Comment:
I think the first sentence "Since a multiple regression involves more then..." could be moved up to the beginning of the "Checking Multicollinearity" section. It would be great to first define the term you have used in the section's title especially since it is described in great detail.